sys/vm/vm_mmap.c

```diff
@@ -631,9 +640,9 @@ kern_mmap_req(struct thread *td, struct mmap_req *mrp)
 		return (EINVAL);
 	}
 	if (prot != PROT_NONE &&
-	    (prot & ~(PROT_READ | PROT_WRITE | PROT_EXEC)) != 0) {
+	    (prot & ~(_PROT_ALL)) != 0) {
 		SYSERRCAUSE("%s: Unexpected protections 0x%x", __func__,
-		    (prot & ~(PROT_READ | PROT_WRITE | PROT_EXEC)));
+		    (prot & ~(_PROT_ALL)));
```

Comment:
Perhaps commit this first as an NFC (and also upstream it) that then facilitates the later commit?

Reply:
This check wasn't upstream and turns out to be a duplicate so into the trash it goes.